apollo-router/src/configuration/migrations/0031-experimental_reuse_query_fragments.yaml

```diff
@@ -0,0 +1,4 @@
+description: supergraph.experimental_reuse_query_fragments is deprecated
```

Comment:
I don't have a context for those changes, but it might be worth mentioning that `supergraph.generate_query_fragments` is enabled by default and is a safe feature. 
I think it would look less scary than just removing the flag.

Reply:
we do have in the [changelog](https://github.com/apollographql/router/releases/tag/v1.58.0), but it's a good call to be extra explicit here.